openlibrary/templates/lib/nav_head.html

```diff
@@ -33,6 +28,35 @@
     </a>
   </div>
 
+  <ul class="navigation-component">
```

Comment:
One thing I'd also suggest, if we can add `id` attributes to the relevant menu buttons (e.g. `id="mobile-hamburger-menu"` - or something, as long as it's unique I'm happy) & then add an `aria-labelledby` attribute that references the value in the relevant menu list. So like:
```
<button id="hamburger">[Hamburger Menu]</button>
...
<ul aria-labelledby="hamburger>[Menu Contents]</ul>
```

Reply:
@bpmcneilly Account dropdown already has an `id="userToggle"` which is not unique as it is used in one more place so what should I do?